testresultparser/junit.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This shouldn't be part of what we share in git. Can you a rule to `.gitignore` to ignore those files from version control?